package/MDAnalysis/core/universe.py

```diff
@@ -68,21 +68,9 @@
 # trigger an MPI warning because importing the uuid module triggers a call to
 # os.fork(). This happens if MPI_Init() has been called prior to importing
 # MDAnalysis. The problem is actually caused by the uuid module and not by
-# MDAnalysis itself. Python 3.7 fixes the problem. However, for Python < 3.7,
-# the uuid module works perfectly fine with os.fork() disabled during import.
-# A clean solution is therefore simply to disable os.fork() prior to importing
-# the uuid module and to re-enable it afterwards.
+# MDAnalysis itself. Python 3.7 fixes the problem.
```

Comment:
We could remove the whole comment if we want, but since we don't test it, it might be good to keep for historical / in case it happens again?

Reply:
I’d keep it.